frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/subscribe-to-ocs.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { nav } from '../../../integration-tests-cypress/views/nav';
+import { primaryButton } from '../../../integration-tests-cypress/views/form';
+import { installSucess } from '../views/olm';
+import { checkErrors } from '../../../integration-tests-cypress/support';
+
+describe('Subscribe to OCS Operator from OperatorHub', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    nav.sidenav.clickNavLink(['Operators', 'OperatorHub']);
+    cy.byTestID('search-operatorhub').type('Openshift Container Storage');
+    cy.byTestID('ocs-operator-redhat-operators-openshift-marketplace').click();
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  it('Subscribes to OCS Operator successfully', () => {
+    cy.byLegacyTestID('operator-install-btn').click({ force: true });
+    cy.byTestID('Operator recommended namespace:-radio-input').should('be.checked');
+    cy.byTestID('enable-monitoring').click();
+    cy.get(primaryButton).click();
+    cy.get(installSucess, { timeout: 180000 }).should('be.visible');
+    cy.exec('oc get project openshift-storage -o yaml')
+      .its('stdout')
+      .should('contain', 'openshift.io/cluster-monitoring: "true"');
+    // Rook, Noobaa and OCS pod should come up after installation.
+    cy.exec('oc get po -n openshift-storage')
+      .its('stdout')
+      .should('contain', 'noobaa-operator')
+      .should('contain', 'ocs-operator')
+      .should('contain', 'rook-ceph-operator');
+  });
```

Comment:
Same comment here. We shouldn't be using substring matches on CLI output. Better to get the JSON and parse it.